packages/actor-scraper/playwright-scraper/INPUT_SCHEMA.json

```diff
@@ -0,0 +1,277 @@
+            "default": "RECOMMENDED",
+            "editor": "select",
+            "enum": [
+                "RECOMMENDED",
+                "PER_REQUEST",
+                "UNTIL_FAILURE"
+            ],
+            "enumTitles": [
+                "Use recommended settings",
+                "Rotate proxy after each request",
+                "Use one proxy until failure"
+            ]
+        },
+        "sessionPoolName": {
+            "title": "Session pool name",
+            "type": "string",
+            "description": "<b>Use only english alphanumeric characters dashes and underscores.</b> A session is a representation of a user. It has it's own IP and cookies which are then used together to emulate a real user. Usage of the sessions is controlled by the Proxy rotation option. By providing a session pool name, you enable sharing of those sessions across multiple actor runs. This is very useful when you need specific cookies for accessing the websites or when a lot of your proxies are already blocked. Instead of trying randomly, a list of working sessions will be saved and a new actor run can reuse those sessions. Note that the IP lock on sessions expires after 24 hours, unless the session is used again in that window.",
+            "editor": "textfield",
+            "minLength": 3,
+            "maxLength": 200,
+            "pattern": "[0-9A-z-]"
+        },
+        "initialCookies": {
+            "title": "Initial cookies",
+            "type": "array",
+            "description": "The provided cookies will be pre-set to all pages the scraper opens.",
+            "default": [],
+            "prefill": [],
+            "editor": "json"
+        },
+        "launcher": {
+            "sectionCaption": "Browser configuration",
+            "title": "Browser Type",
+            "description": "Choose the browser to launch.",
+            "type": "string",
+            "editor": "select",
+            "enum": [
+                "chromium",
+                "firefox"
+            ],
+            "enumTitles": [
+                "Chromium",
+                "Firefox"
+            ],
+            "prefill": "chromium",
+            "default": "chromium"
+        },
+        "useChrome": {
+            "title": "Use Chrome (could only be used if Chromium Browser is selected)",
```

Comment:
Maybe too long for a title? But the explanation in parentheses is absolutely crucial too.

Reply:
It looks ok as it's just a boolean, and there's a whole screen empty) 👌